BT-Tracking/AppDelegate.swift

```diff
@@ -200,7 +200,7 @@ private extension AppDelegate {
             })
         } else {
             // User with phone number is old user
-            if Auth.auth().currentUser != nil {
+            if Auth.auth().currentUser?.phoneNumber != nil {
```

Comment:
Testoval jsi to proti erousce 1.3? Me to teda nefungovalo. Ale nechal jsem tam ten comment, trochu chybne...

Reply:
V pondeli to otestuju poradne. Mam pocit ze v simulatoru mi to taky nefungovalo, ale na zarizeni ano.